packages/@webex/internal-plugin-team/src/team.js

```diff
@@ -187,13 +188,32 @@ const Team = WebexPlugin.extend({
    * Join a team conversation
    * @param {TeamObject} team
    * @param {ConversationObject} conversation
+   * @param {userId} userId
    * @returns {Promise} Resolves with the conversation
    */
-  joinConversation(team, conversation) {
+  joinConversation(team, conversation, userId) {
+    const body = {
```

Comment:
can we only do this if the userID is present , i am not sure if widgets are using this